tests/main/classic-custom-device-reg/task.yaml

```diff
@@ -71,13 +71,13 @@ execute: |
     echo "Wait for seeding to be done"
     while ! snap changes | grep -q "Done.*Initialize system state"; do sleep 1; done
     echo "We have a model assertion"
-    snap known model|MATCH "model: my-classic-w-gadget"
+    snap model --assertion | MATCH "model: my-classic-w-gadget"
 
     echo "Wait for device initialisation to be done"
     while ! snap changes | grep -q "Done.*Initialize device"; do sleep 1; done
 
     echo "Check we have a serial"
-    snap known serial|MATCH "authority-id: developer1"
-    snap known serial|MATCH "brand-id: developer1"
-    snap known serial|MATCH "model: my-classic-w-gadget"
-    snap known serial|MATCH "serial: 7777"
+    snap model --serial --assertion|MATCH "authority-id: developer1"
```

Comment:
@pedronis sorry I didn't get back in time to respond to your ping over IRC yesterday, but this is the use case for showing authority-id I was referring to, as I said it would be nice if in our own tests we could only use `snap model --verbose` instead of needing to resort to `snap model --assertion`. The use case in this PR is for our tests, but I think in general having as much as possible from the assertion exposed in a format that is nicely consumed from scripts is useful. 

I admit to not knowing of an explicit use case outside of tests that one would need to know the authority-id for a model assertion though.